### DISCUSSION TITLE
How does PocketHost maintain the platform? How do they earn money?

### DISCUSSION THREAD

**Discussion by raldincasidar-studio** [pockethost/pockethost#123] [Q&A]
I want to host a project with PocketHost and I'm wondering if this is reliable and will not be down.

**Comment by benallfree** [comment]
Hello, thanks for asking and joining in the discussion.

PocketHost is a personal project managed entirely by me. I created it as a service to the PocketBase community because I love the sovereignty principle but also don’t always want to self host. So I created an entire multitenant hosting platform. Ironically, that is more work than setting up self hosting myself but now at least others can benefit too. I use PocketHost for my projects too. 

The features and priorities in development are balanced between my own needs and things that would obviously be needed if I didn’t have direct access to the server. Things like backing up and restoring instances from a command line utility, for example. 

I’ve already committed enough of my personal funds in an endowment that will keep PocketHost running for the next 10 years if needed. Depending on how it grows, it might turn into a nonprofit company. No need for now. 

I have been thinking thinking about a paid tier, but there will always be a free tier. I think most projects won’t need more than the free tier.

Other options for future revenue include enterprise setups, customization, priority support, etc. I think there are lots of options for revenue if the service grows to a point where that makes sense.

For now, I would invite you to enjoy PocketHost with the understanding that you can always export your instance data and self host if things ever change.
